doc_src/fish_indent.txt

```diff
@@ -7,12 +7,16 @@ fish_indent [OPTIONS]
 
 \subsection fish_indent-description Description
 
-`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output.
+`fish_indent` is used to indent a piece of fish code. `fish_indent` reads commands from standard input and outputs them to standard output or a specified file.
 
 The following options are available:
 
 - `-d` or `--dump` dumps information about the parsed fish commands to stderr
 
+- `-w` or `--write` indents a specified file and immediatly writes to that file
```

Comment:
I think it would be useful to be able to write to the same file that was specified in some way without having to redirect to it. I am thinking that this could be used in places where redirections is not really convenient, like some editor or tool using `fish_indent` to auto-format the code.